src/doc/__tests__/DocHighlightThread-test.js

```diff
@@ -319,12 +319,9 @@ describe('doc/DocHighlightThread', () => {
         const pageEl = {
             getBoundingClientRect: jest.fn().mockReturnValue({ height: 0, top: 10 }),
         };
-        let quadPoint = {};
 
         beforeEach(() => {
-            quadPoint = {};
-            thread.location.quadPoints = [quadPoint, quadPoint, quadPoint];
-
+            thread.location.quadPoints = [[], [], []];
```

Comment:
Should this be an array of `Objects`?

Reply:
Nope, it's an array of arrays. The test was just wrong before.